lib/Backend/Encoder.cpp

```diff
@@ -295,8 +295,9 @@ Encoder::Encode()
     {
         if (m_func->IsOOPJIT())
         {
-            Js::ThrowMapEntry * throwMap = NativeCodeDataNewArrayNoFixup(m_func->GetNativeCodeDataAllocator(), Js::ThrowMapEntry, m_pragmaInstrToRecordMap->Count());
-            for (int32 i = 0; i < m_pragmaInstrToRecordMap->Count(); i++)
```

Comment:
The calls to m_pragmaInstrToRecordMap->Count() for  AllocatorArray and loop counter is not recognized as equivalent by the analyzer.